src/index.html

```diff
@@ -6,13 +6,61 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="/src/styles/index.scss"
```

Comment:
The `href` attribute for the stylesheet should be checked to ensure it points to the correct path. Since this is an HTML file, the path should be relative to the location of this file. If `index.scss` is compiled to `index.css`, make sure to link to the CSS file instead.